scripts/functions/requirements/osx_brew

```diff
@@ -318,6 +318,12 @@ requirements_osx_brew_define_openssl()
       brew_openssl_package="openssl"
       ;;
 
+    (ruby-2*|ruby-3.0*)
```

Comment:
I believe this maybe should be `... ruby-3*)`
?  

for me this did not work installing ruby 3.3.4

Reply:
Did you tried running `rvm get master` before you tried to install ruby?